templates/galaxy/config/staging_job_conf.yml.j2

```diff
@@ -35,60 +35,10 @@ execution:
       function: map_tool_to_destination
       rules_module: tpv.rules
       tpv_config_files:
-        - "{{ galaxy_dynamic_job_rules_dir }}/total_perspective_vortex/vortex_config.yml"
-    gateway:
-      runner: dynamic
-      type: python
-      function: gateway
-    dynamic_dtd:
-      runner: dynamic
-      type: dtd
-    slurm:
-      runner: slurm
-      nativeSpecification: "--nodes=1 --ntasks=2 --ntasks-per-node=2 --mem=7760"
-      singularity_enabled: false
-      singularity_volumes: '{{ slurm_singularity_volumes }}'
-      singularity_default_container_id: '{{ singularity_default_container_id }}'
-      docker_enabled: false
-      docker_sudo: false
-      docker_volumes: '{{ slurm_docker_volumes }}'
-    pulsar:
-      runner: pulsar_au_01
-      jobs_directory: /mnt/pulsar/files/staging
-      transport: curl
-      remote_metadata: 'false'
-      default_file_action: remote_transfer
-      outputs_to_working_directory: false
-      dependency_resolution: remote
-      rewrite_parameters: 'true'
-      persistence_directory: /mnt/pulsar/files/persisted_data
-      submit_native_specification: "--nodes=1 --ntasks=2 --ntasks-per-node=2"
-      docker_enabled: false
-      docker_sudo: false
-      docker_volumes: '{{ pulsar_docker_volumes }}'
-      docker_run_user: '1000'
-      singularity_enabled: false
-      singularity_volumes: '{{ pulsar_singularity_volumes }}'
-      singularity_default_container_id: '{{ singularity_default_container_id }}'
-      container_resolvers: 
-        - type: explicit_singularity
-        - type: mulled_singularity
-      env:
-        - name: SINGULARITY_CACHEDIR
-          value: /mnt/pulsar/deps/singularity
-        - name: SINGULARITY_TMPDIR
-          value: /mnt/pulsar/deps/singularity/tmp
-    interactive_pulsar:
-      runner: pulsar_embedded
-      outputs_to_working_directory: false
-      docker_enabled: true
-      docker_volumes: $defaults
-      docker_sudo: false
-      docker_net: bridge
-      docker_auto_rm: true
-      docker_set_user: ''
-      require_container: true
-      container_monitor_result: callback
+        - "{{ galaxy_dynamic_job_rules_dir }}/total_perspective_vortex/staging_tools.yml"
```

Comment:
Shouldn't there be a reference to the shared database and production tools file?

Reply:
Good point, I will add the shared database

staging_tools.yml is a symlink (in the infrastructure repo) to the production tool file, so copies to the staging VM with the same name.  The way that the galaxyproject.galaxy role handles job rules files makes it difficult to list a file in a different directory, hence the symlink.  staging_vortex_config.yml defines the default tool for staging with weighted random sampling as the rank function.